docs/v3/core/v3.0.rst

```diff
@@ -1042,8 +1086,13 @@ chunking as above, but using an extension data type::
         "chunk_grid": {
             "name": "regular",
             "configuration": {
-              "chunk_shape": [1000, 100],
-              "separator" : "/"
+              "chunk_shape": [1000, 100]
```

Comment:
Just noticed this since you fixed it already above:

```suggestion
                "chunk_shape": [1000, 100]
```

Reply:
Done.